### DISCUSSION TITLE
is there a reason why windows arm gdc doesnt have TUI while linux version does

### DISCUSSION THREAD

**Discussion by EdwinFairchild** [xpack-dev-tools/arm-none-eabi-gcc-xpack#36] [Q&A] [answered]
Just wondering if there is a reason, I mostly use it on linux and love it but on windows it does not seem to be enabled.
If im honest im not even sure if the versions I use in linux I got from here or elsewhere haha

**Comment by ilg-ul** [comment, marked as answer]
Yes, some of the dependencies required could not be compiled for windows.

If you find a distribution that includes tui on windows, please let me know and I'll reevaluate.

**Reply by EdwinFairchild** [reply]
Thats fair, I was just wondering if it was a matter of not many people wanting it or simply not possible without unreasonable effort. I dont mind it just being on linux.

**Comment by ilg-ul** [comment]
After some tweaking, I managed to compile the latest version of ncurses on Windows and the latest gdb with TUI support.

Please check the experimental binaries from:

- https://github.com/xpack-dev-tools/pre-releases/releases/tag/experimental
